library-best-practices/JLBP-8.md

```diff
@@ -5,12 +5,13 @@ Libraries at 0.x versions and unstable features (which should be marked with
 annotations if they live in a stable library, as outlined in
 [JLBP-3](JLBP-3.md)) make no promises about stability. If they become widely
 used, they can cause problems when there are surface breakages (which is allowed
-per the rules of unstable functionality, explained in [JLBP-3](JLBP-3.md)). If
-such breakages can cause widespread pain, the unstable features can become
-"common-law GA." The preferable path is to promote such widely-used unstable
-libraries and features to stability, even if the functionality is not complete,
-because stability is not about completeness, it is about guarantees to not break
-users frequently or without warning.
+per the rules of unstable functionality, explained in [JLBP-3](JLBP-3.md)).
+If such breakages can cause widespread pain, the unstable features can be recognized
+as general availability (GA) by many users.
```

Comment:
@garrettjonesgoogle I didn't get the meaning of "common-law GA". So I tried to simplified it. Does this change still reflect what you meant?
(Still I don't think my sentence is not having correct logic around "if such breakages can cause... ")

Reply:
We can reword but I think the original is clearer to me.

Reply:
"common-law GA" is on oblique reference to https://en.wikipedia.org/wiki/Common-law_marriage .

Reply:
I didn't correctly interpret the sentence 'If such breakages ... can become "common-law GA"'. And thus the previous suggestion was incorrect.